User-Commands.md

```diff
@@ -33,20 +33,24 @@ ___
 Opens a web interface to edit permissions for the specified group. After changes are saved, a command will be given that you need to run for the changes to take effect.
 
 ___
-#### `/lp user <user> promote <track> [context...]`  
+#### `/lp user <user> promote <track> [context...] [--dont-add-to-first] [-s]`  
 **Permission**: luckperms.user.promote  
 **Arguments**:  
 * `<track>` - the track to promote along
 * `[context...]` - the [contexts](Context) to promote in
+* `[--dont-add-to-first]` - the command will fail to run if the user doesn't have parent groups on said track
+* `[-s]` - silent, don't print the user progress on the track
 
-This command will promote a user along a track. Firstly, the command will check to see if the user is on the track specified in the given contexts. If the user is not on the track, they will be added to the first group on the track. If they are on the track in more than one place, the command will fail. In all other cases, the user will be promoted up the track, and will be removed from the existing group. If the track action affects their primary group, that will be updated too.
+This command will promote a user along a track. Firstly, the command will check to see if the user is on the track specified in the given contexts. If the user is not on the track, they will be added to the first group on the track unless `--dont-add-to-first` is specified, if it is the command will fail. If they are on the track in more than one place, the command will fail. In all other cases, the user will be promoted up the track, and will be removed from the existing group. If the track action affects their primary group, that will be updated too.
```

Comment:
maybe have a section for flags and their function instead of making the paragraph bigger